main.go

```diff
@@ -129,8 +130,7 @@ func main() {
 		Scheme: scheme,
 		// HEADS UP: once controller runtime is updated and this changes to metrics.Options{},
 		// and in case you configure TLS / SecureServing, disable HTTP/2 in it for mitigating related CVEs!
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Metrics:                metricsServer.Options{BindAddress: "0"},
```

Comment:
- why don't you use metricsAddr anymore?
- have a look at the comment above! (see NHC as example)

Reply:
Too much copy-paste. Good catch